packages/react-scripts/package.json

```diff
@@ -64,6 +64,7 @@
     "url-loader": "0.5.7",
     "webpack": "1.13.2",
     "webpack-dev-server": "1.16.2",
+    "webpack-env-define-plugin": "../webpack-env-define-plugin",
```

Comment:
Don't reference a local path. Refer to the package the way it would appear on npmjs.com

See https://github.com/lerna/lerna for more details on how these packages link together.